keywrap/pkcs11/keywrapper_pkcs11.go

```diff
@@ -0,0 +1,233 @@
+/*
+   Copyright The ocicrypt Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package pkcs11
+
+import (
+	"github.com/containers/ocicrypt/config"
+	"github.com/containers/ocicrypt/keywrap"
+	"github.com/miekg/pkcs11"
+	"github.com/pkg/errors"
+	"strings"
+)
+
+var (
+	KeyLabel   = "imgenc"
+	OAEPParams = &pkcs11.OAEPParams{
+		HashAlg:    pkcs11.CKM_SHA_1,
```

Comment:
SHA256 would be better at this point.

Reply:
I'll try to use SHA256 with softhsm2. testing before that have some error. so i rollback to SHA1

Reply:
can we keep SHA1 for experimental, because softhsm2 not support OAEP with SHA256. maybe can change to default with SHA256. [softhsm2 code](https://github.com/opendnssec/SoftHSMv2/blob/827ab25e9d5c571502eaf7900a5cc1732223240f/src/lib/SoftHSM.cpp#L12395-L12403)

Reply:
Yes, just found out about this also...